docs/cck/components/Combat System.md

```diff
@@ -0,0 +1,59 @@
+# Combat System
+Adding this component enables PvP and PVE style gamemodes in your world. Best used alongside the **Damage** Component and the **Game Instance Controller** Component
+
+### Properties
+
+##### Reference ID
+
+##### Health
++ **Health Base Amount:** This is the default Health each player starts with at the begining of a Game or Round.
++ **Health Max Amount:** This is the Maximum amount of Health a player can have at any given time.
+
+##### Health Regeneration
++ **Health Regeneration Delay:** The time in seconds before the player starts regenerating Health.
++ **Health Regeneration Rate:** How much Health the player receives per second.
```

Comment:
This is worded strangely. Would "The amount of health the player recovers per second." work better here?

Reply:
will reword this entry with your changes.